src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,8 @@
 
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T> {
```

Comment:
```suggestion
public interface MachineProducer<T exdends Machine> {
```
I think it is better to limit the type of parameterization. What do you think?

Reply:
Yeah good point. Thanx)